SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/packages.config

```diff
@@ -15,7 +15,7 @@
   <package id="System.Diagnostics.Debug" version="4.0.11" targetFramework="net452" />
   <package id="System.Diagnostics.Tools" version="4.0.1" targetFramework="net452" />
   <package id="System.Diagnostics.Tracing" version="4.1.0" targetFramework="net452" />
-  <package id="System.Globalization" version="4.0.11" targetFramework="net452" />
```

Comment:
@jwallra @achocron how will that impact customers? Do we need to update nuspec dependencies?